tracker/storage.go

```diff
@@ -60,23 +60,31 @@ func InitStorage(dbName string) *Storage {
   _, err2 := db.Exec(query)
   checkErr(err2)
 
-  // Prepare Add Statement
-  query =
-    "INSERT INTO " + DB_TABLE_NAME + "(" +
-      DB_COLUMN_EVENT +
-    ") values(?);"
-  addStmt, err3 := db.Prepare(query)
-  checkErr(err3)
+  return &Storage{ DbName: dbName }
+}
 
-  return &Storage{ DbName: dbName, DbConn: db, AddStmt: addStmt }
+func getDbConn(dbName string) (*sql.DB, error) {
```

Comment:
Please consider putting the checkErr(err) in the getDbConn. 
That way we don't need to return error.

Reply:
Good idea!